interfaces/HO-Portal/src/app/program/program.component.html

```diff
@@ -41,20 +42,23 @@
         <app-metrics [program]="program"></app-metrics>
 
       </ion-col>
+
       <ion-col size=9>
 
+        <app-program-people [selectedPhase]="selectedPhase"
+                      [programId]="program?.id"
+                      [userRole]="currentUserRole"
+                      (emitCompleted)="emitPeopleCompleted($event)"
```

Comment:
Re: renaming the EventEmitter-properties:

This would then become:
```
(isCompleted)="onPeopleCompleted($event)"
```

So they both NOT include the 'emit' verb. Because the action they perform is not the same. (only 1 emits a thing actually).